modules/sdk-core/src/bitgo/baseCoin/baseCoin.ts

```diff
@@ -267,6 +267,14 @@ export abstract class BaseCoin implements IBaseCoin {
     return false;
   }
 
+  /**
+   * Check whether a coin supports lightning transactions
+   * @returns {boolean}
+   */
+  supportsLightning() {
```

Comment:
I wonder if we can replace this with more generic `getFeatures()`/`hasFeature()` functions that would look at the `features` property of the underlying statics coin definition. Open to thoughts on this.

Reply:
I also looked into `CoinFeature` , but it seems only account based coins are making use of it. I might be missing something, but I am also open to thoughts on using features